frontend/pages/signup.tsx

```diff
@@ -149,31 +156,45 @@ export default function Signup() {
   };
 
   return (
-    <Layout
-      title={texts.sign_up}
-      isLoading={isLoading}
-      message={errorMessage}
-      messageType={errorMessage && "error"}
-    >
-      {curStep === "basicinfo" ? (
-        <BasicInfo
-          values={userInfo}
-          handleSubmit={handleBasicInfoSubmit}
-          errorMessage={errorMessages[steps[0]]}
-        />
-      ) : (
-        curStep === "personalinfo" && (
-          <AddInfo
-            values={userInfo}
-            handleSubmit={handleAddInfoSubmit}
-            errorMessage={errorMessages[steps[1]]}
-            handleGoBack={handleGoBackFromAddInfo}
-            locationInputRef={locationInputRef}
-            locationOptionsOpen={locationOptionsOpen}
-            handleSetLocationOptionsOpen={handleSetLocationOptionsOpen}
-          />
-        )
-      )}
-    </Layout>
+    <WideLayout message={errorMessage} messageType="error" isLoading={isLoading}>
+      <Container maxWidth={hugeScreen ? "xl" : "lg"}>
+        <ThemeProvider theme={themeSignUp}>
+          <ContentImageSplitView
+            minHeight="80vh"
+            content={
```

Comment:
ContentImageSplitView displays content and an image next to each other. As these are two children, they have to be inputed as props